pom.xml

```diff
@@ -27,8 +27,8 @@
 		<jaeger.version>1.1.0</jaeger.version>
 		<opentracing.version>0.33.0</opentracing.version>
 		<opentracing-kafka-client.version>0.1.12</opentracing-kafka-client.version>
-		<micrometer.version>1.3.1</micrometer.version>
-		<jmx-prometheus-collector.version>0.12.0</jmx-prometheus-collector.version>
+		<micrometer.version>1.3.9</micrometer.version>
+		<jmx-prometheus-collector.version>0.10</jmx-prometheus-collector.version>
```

Comment:
Are we downgrading to older version here? Why? Or is that again some weird numbering issue in JMX Exporter?

Reply:
To let it match a productized version.

Reply:
I don't think we normally do it ... If it is just dependency change, we can also do it directly in productisation and keep the newer version here I guess?

Reply:
reverted this change, and we can do it in the productized version.